gui/wxpython/gui_core/mapdisp.py

```diff
@@ -160,17 +160,29 @@ def GetToolSwitcher(self):
 
     def SetProperty(self, name, value):
         """Sets property"""
-        if self.HasProperty("projection"):
-            self.statusbarManager.SetProperty(name, value)
-        else:
+        if name == "projection":
```

Comment:
This is a temporary solution since we need self.mapFrane.GetProperty("projection") in some statusbar classes such as SbGoTo and so on...